semgrep/README.md

```diff
@@ -1,17 +0,0 @@
-# semgrep-cli Development
-
-Semgrep uses pipenv to manage a virtual environment:
-
-```
-pipenv install --dev
```

Comment:
Is this information stored elsewhere now?

Reply:
And testing: https://github.com/returntocorp/semgrep/tree/develop/doc#testing